packages/twenty-server/src/workspace/messaging/services/is-person-email.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class IsPersonEmailService {
+  constructor() {}
+
+  isPersonEmail(email: string | undefined): boolean {
+    if (!email) return false;
+
+    const nonPersonalPatterns = [
+      /noreply/,
+      /no-reply/,
+      /do_not_reply/,
+      /no.reply/,
+      /^accounts@/,
+      /^info@/,
+      /^admin@/,
+      /^contact@/,
+      /^hello@/,
+      /^support@/,
+      /^sales@/,
+      /^feedback@/,
+      /^service@/,
+      /^help@/,
+      /^mailer-daemon/,
+      /^notifications?/,
+      /^digest/,
+      /^auto/,
+      /^apps/,
+      /^assign/,
+      /^comments/,
+      /^customer-success/,
+      /^enterprise/,
+      /^esign/,
+      /^express/,
+      /^forum/,
+      /^gc@/,
+      /^learn/,
+      /^mailer/,
+      /^marketing/,
+      /^messages/,
+      /^news/,
+      /^notification/,
+      /^payments/,
+      /^receipts/,
+      /^recrutement/,
+      /^security/,
+      /^service/,
+      /^support/,
+      /^team/,
+    ];
+
+    for (const pattern of nonPersonalPatterns) {
```

Comment:
nit: You can avoid a forloop with this regular expression (introducing OR | )
```
const nonPersonalPattern = /^(noreply|no-reply|do_not_reply|no\.reply|accounts@|info@|admin@|contact@|hello@|support@|sales@|feedback@|service@|help@|mailer-daemon|notifications?|digest|auto|apps|assign|comments|customer-success|enterprise|esign|express|forum|gc@|learn|mailer|marketing|messages|news|notification|payments|receipts|recrutement|security|service|support|team) /;

if (nonPersonalPattern.test(email)) {
  return false;
}

return true;
```
I think it's a bit negligible in our case here so up to you